next.config.js

```diff
@@ -29,4 +29,15 @@ module.exports = {
     };
     return config;
   },
+  // images: {
```

Comment:
You can remove the code. This is only needed when we have a remote image server.